### PR TITLE
Use xz to compress rootfs

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

File: docs/build.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ A convenience `build` script is included that builds the image and runs basic te
 
 ### Builder
 
-The image is built using a builder Docker container based on the `debian` image. This builder image lives in the `builder` sub-directory of the project and uses a `mkimage-alpine.sh` script to generate an Alpine Linux `rootfs.tar.gz` file. This file then gets copied to the root of the project so we can build the main Alpine Linux image by just using the `ADD` command to automatically untar the components to the resulting image.
+The image is built using a builder Docker container based on the `debian` image. This builder image lives in the `builder` sub-directory of the project and uses a `mkimage-alpine.sh` script to generate an Alpine Linux `rootfs.tar.zz` file. This file then gets copied to the root of the project so we can build the main Alpine Linux image by just using the `ADD` command to automatically untar the components to the resulting image.
```
Comment (ncopa) [inline docs/build.md]: looks like `.zz` is a typo here

Comment (PeterDaveHello) [inline docs/build.md]: Fixed now, thanks @ncopa

**Comment by PeterDaveHello** [comment]
Sure, I'll do it.

**Comment by PeterDaveHello** [comment]
Done.

**Comment by andyshinn** [comment]
But what is the reasoning? And also, will users need `xz` available for the build script?

**Comment by PeterDaveHello** [comment]
xz should provide better compression ratio

**Comment by tianon** [comment]
xz support is also built into Docker itself; I compress the Debian rootfs
tarballs with it since it gives me smaller sizes (not as compelling an
argument for the tiny Alpine tarballs)

**Comment by jumanjiman** [comment]
Here is a handful of informational references about xz:

* https://www.kernel.org/doc/Documentation/xz.txt
* https://en.wikipedia.org/wiki/Xz
* https://unix.stackexchange.com/questions/108100/why-are-tar-archive-formats-switching-to-xz-compression-to-replace-bzip2-and-wha

(i don't advocate either way on this)

**Pull request merged**